genai-perf/genai_perf/inputs/converters/base_converter.py

```diff
@@ -64,3 +64,9 @@ def _select_model_name(self, config: InputsConfig, index: int) -> str:
             raise GenAIPerfException(
                 f"Model selection strategy '{config.model_selection_strategy}' is unsupported"
             )
+
+    def _add_request_params(
```

Comment:
This gets reused among converters, so I moved it to the base converter class. Other converts can reimplement it for custom logic.